pipelines/models/tensorflow-housing/README.md

```diff
@@ -0,0 +1,24 @@
+
+# How was this model generated?
+
+## Data source
+
+The model uses
+https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing,
+see
+https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_california_housing.html
+for more information about the data.
+
+## Generating model
+
+With https://github.com/mlflow/mlflow checked out and specifically
```

Comment:
Do you plan to change this to use the steps you mentioned in a comment with the ubi9 container rather than cloning the repo, or are you going to to leave it as it is here for now?

Reply:
I haven't decided yet. I hope to still have some discussion about the `pyenv` vs. UBI python approach.

I'm currently thinking about having "How it was generated" section show the `git clone` but keeping it vague as to what exactly was used. And then have section like "Possible way to retrain the model" where I'd show the direct UBI container approach as that looks very cool.

Reply:
Sounds good to me :+1: